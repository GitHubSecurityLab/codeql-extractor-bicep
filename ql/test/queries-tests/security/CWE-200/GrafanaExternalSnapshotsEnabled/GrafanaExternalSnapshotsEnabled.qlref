security/CWE-200/GrafanaExternalSnapshotsEnabled.ql
