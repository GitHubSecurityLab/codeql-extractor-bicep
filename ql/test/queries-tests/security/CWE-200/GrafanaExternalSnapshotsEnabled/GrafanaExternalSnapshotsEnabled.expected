| app.bicep:11:18:13:7 | Snapshots | External snapshots are enabled in Grafana configuration, which could lead to unintended sharing of dashboard data with external services. |
